engine/table/src/test/java/io/deephaven/engine/util/WorkerPythonEnvironment.java

```diff
@@ -24,6 +24,7 @@
  * remote Python session should execute queries which interact wtih this class. The script sessions that run for
  * PersistentQueries or consoles are handled separately by the {@link PythonDeephavenSession}.
  */
+@Deprecated
```

Comment:
same as above

Reply:
Removed - that said, it would have resolved cleanly with you move.